metrics/src/metrics.rs

```diff
@@ -405,9 +405,23 @@ fn get_metrics_config() -> Result<MetricsConfig, String> {
     if !config.complete() {
         return Err("SOLANA_METRICS_CONFIG is incomplete".to_string());
     }
+
     Ok(config)
 }
 
+pub fn metrics_config_sanity_check(cluster_type: ClusterType) -> Result<(), String> {
+    let config = get_metrics_config()?;
```

Comment:
it should be ok if metrics aren't configured at all, right?  might be time for a proper error type here (separate pr?) rather than the lazy `Result<T, String>`. should be pretty easy to swap out with an enum and `thiserror` crate